ovirt/resource_ovirt_vm.go

```diff
@@ -636,6 +684,18 @@ func resourceOvirtVMCreate(d *schema.ResourceData, meta interface{}) error {
 		}
 	}
 
+	// update the VM with the auto pinning
+	if isAutoPinning {
+		autoPinningPolicy := ovirtsdk4.AutoPinningPolicy(fmt.Sprint(d.Get("auto_pinning_policy")))
+		if autoPinningPolicy != ovirtsdk4.AUTOPINNINGPOLICY_DISABLED {
+			log.Printf("[DEBUG] Setting Auto Pinning Policy to VM (%s).", d.Id())
+			err := ovirtUpdateAutoPinning(d.Id(), autoPinningPolicy, meta)
+			if err != nil {
+				log.Printf("[DEBUG] Updating the VM (%s) with auto pinning policy failed!", d.Id())
```

Comment:
you should probably print the error and return a meaning full error from ovirtUpdateAutoPinning